recipes-devtools/rust/rust-version.inc

```diff
@@ -1,6 +1,6 @@
 # Note: if you adjust this, you'll also need to change the hashes in
 # rust-source.bb
 SOURCE_NAME = "rust"
-PV = "1.10.0"
+PV = "1.12.0"
```

Comment:
I should also note I haven't made any changes. I just wanted to test another branch tickling GitHub + Jenkins in the right way.